src/ucp/api/ucp.h

```diff
@@ -168,6 +172,10 @@ enum ucp_dt_type {
                                                                 mask */
 };
 
+struct ucp_worker_params {
+    cpu_set_t                     cpu_set; /* cpu_set affinity to use when creating a cq */
```

Comment:
I will change it once a new issue is opened and a new PR is completed - about parameters